charmbracelet/wishlist
# feat: allow discovery of mDNS services

this builds on #48, adding the discovered nodes as "seed" entrypoints to `sshparse`, which will in turn apply `Host` rules as expected.


closes #48 
refs #135
cc/ @wizardishungry 


---

to try it out:


create with `/etc/avahi/services/ssh.service`:

```xml
<?xml version="1.0" standalone="no"?>
<!DOCTYPE service-group SYSTEM "avahi-service.dtd">
<service-group>   
  <name replace-wildcards="yes">%h</name>
  <service>
    <type>_ssh._tcp</type>
    <port>22</port>
  </service>
</service-group>
```

then 

```sh
service avahi-daemon restart

# to test:
avahi-browse -a
```

you can then run wishlist with `-z` and it should load all `_ssh._tcp` services accessible...
